File: C/1013.c
#include <stdio.h>

int main() {
    int a, b, c, maior, maiorAb;

    scanf("%d %d %d", &a, &b, &c);

    maiorAb = (a + b + abs(a - b)) / 2;
    maior = (maiorAb + c + abs(maiorAb - c)) / 2;
    printf("%d eh o maior\n", maior);

    return 0;
}

File: Python/1013.py
a = int(input())
b = int(input())
c = int(input())

maiorAb = (a + b + abs(a - b)) / 2
maior = (maiorAb + c + abs(maiorAb - c)) / 2

print('{:.0f} eh o maior'.format(maior))
